src/index.css

```diff
@@ -90,6 +90,17 @@ main {
   gap: 15rem;
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225287628-d43c7bac-2a39-42f4-bc90-ef046c2f8dd8.png)

- [ ] It would be nice if you remove these sections from your readme file since you have nothing in these sections 👍 Kindly remove them to make your readme file clean and descriptive 👍 Once you have data for these sections then you can add these sections again in your readme file with data 😊